packages.yml

```
@@ -1,6 +1,3 @@
 packages:
   - package: fivetran/hubspot_source
     version: [">=0.11.0", "<0.12.0"]
```

This needs to point at the latest version of the hubspot source package
```suggestion
    version: [">=0.12.0", "<0.13.0"]
```

---

updated